<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoiceSettings</class>
 <widget class="QWidget" name="VoiceSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>In-world voice</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="1" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Playback buffer size</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSlider" name="playbackBufferSlider">
                <property name="maximum">
                 <number>1000</number>
                </property>
                <property name="singleStep">
                 <number>10</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickInterval">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Encode quality</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSlider" name="encodeQualitySlider">
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="singleStep">
                 <number>5</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickInterval">
                 <number>5</number>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Default voice mode</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Michrophone level</string>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QSlider" name="microphoneLevelSlider">
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="singleStep">
                 <number>5</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickInterval">
                 <number>5</number>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QLabel" name="encodeQualityLabel">
                <property name="text">
                 <string>50 %</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="playbackBufferSizeLabel">
                <property name="text">
                 <string>200 ms</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QComboBox" name="defaultVoiceMode"/>
                </item>
               </layout>
              </item>
              <item row="7" column="3">
               <widget class="QLabel" name="microphoneLevelLabel">
                <property name="text">
                 <string>45 %</string>
                </property>
               </widget>
              </item>
              <item row="7" column="4">
               <widget class="QPushButton" name="testMicrophoneButton">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Local test mode</string>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QPushButton" name="applyPlaybackBufferSizeButton">
                <property name="text">
                 <string>Apply</string>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>10</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="4">
               <widget class="QPushButton" name="applyEncodeQualityButton">
                <property name="text">
                 <string>Apply</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Positional audio</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QCheckBox" name="positionalAudioCheckBox">
                <property name="text">
                 <string>enabled</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
